From 7e75336bcf17e76c808ea058a5375f0c0ec005c8 Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Fri, 6 Mar 2020 00:49:10 -0800
Subject: [PATCH 1259/1852] dt-bindings: display: xlnx: mixer: Add rev 5.0 to
 compatible string

commit 5aaa35ba9381d28fe8344dd8fd61cf187fe45d80 from
https://github.com/Xilinx/linux-xlnx.git

This patch updates the compatible string and colorimetry coefficient
entry to support 5.0 IP version. The mixer 5.0 IP introduces config
registers to program the coefficients for color space conversion.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/display/xlnx/xlnx,mixer.txt          | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
index 20f4cec27175..a0b2fc05d5aa 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,mixer.txt
@@ -9,18 +9,21 @@ The Video Mixer always has one streaming input layer, known as master layer.
 
 Required properties:
  - compatible: Must contain atleast one of
+	"xlnx,mixer-5.0" (MIXER 5.0 version)
 	"xlnx,mixer-4.0" (MIXER 4.0 version)
 	"xlnx,mixer-3.0" (MIXER 3.0 version)
  - reg: Base address and size of the IP core.
  - interrupts: Interrupt number.
  - interrupts-parent: phandle for interrupt controller.
  - reset-gpio: gpio to reset the mixer IP
+ - xlnx,enable-csc-coefficient-register: denotes colorimetry
+   coefficients can be programmed, valid for mixer 5.0 version only.
  - xlnx,dma-addr-width: dma address width, valid values are 32 and 64
  - xlnx,bpc: bits per component for mixer
  - xlnx,ppc: pixel per clock for mixer
  - xlnx,num-layers: Total number of layers (excluding logo)
    Value ranges from 1-9 for compatible string xlnx,mixer-3.0 and
-   Value ranges from 1-17 for comptaible string xlnx,mixer-4.0
+   Value ranges from 1-17 for comptaible string xlnx,mixer-4.0 and above
  - layer_[x]: node for [x] layer
  - xlnx,layer-id: layer identifier number
  - xlnx,vformat: video format for layer. See list of supported formats below.
-- 
2.31.1

